### PR TITLE
ROX-15287: fix ocp 4.6 for release-3.74 branch

### PR DESCRIPTION
**Pull request by BradLugo**
/cc @gavin-stackrox 
/assign @BradLugo

### CODE REVIEW

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73__ocp-4-6.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- as: ui-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      TEST_SUITE: ocp-ui-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
```
Comment (BradLugo) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73__ocp-4-6.yaml:53]: not sure if we need this

**Comment by gavin-stackrox** [comment]
Changed files are for `3.73`?

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Review by gavin-stackrox — APPROVED** [review]
/lgtm

**Pull request merged**